Tasks/ContainerStructureTestV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,16 @@
+{
+  "loc.friendlyName": "Container Structure Test",
+  "loc.helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=613742)",
+  "loc.description": "Runs Container structure tests using Google container structure test framework",
```

Comment:
Change to "Uses container-structure-test (https://github.com/GoogleContainerTools/container-structure-test) to validate the structure of an image based on four categories of tests - command tests, file existence tests, file content tests and metadata tests"

Reply:
Done